shared/controller-api/src/main/java/io/pravega/shared/controller/event/ControllerEventSerializer.java

```diff
@@ -44,7 +44,8 @@ protected void declareSerializers(Builder builder) {
                .serializer(DeleteTableEvent.class, 10, new DeleteTableEvent.Serializer())
                .serializer(CreateReaderGroupEvent.class, 11, new CreateReaderGroupEvent.Serializer())
                .serializer(DeleteReaderGroupEvent.class, 12, new DeleteReaderGroupEvent.Serializer())
-               .serializer(UpdateReaderGroupEvent.class, 13, new UpdateReaderGroupEvent.Serializer());
+               .serializer(DeleteScopeEvent.class, 13, new DeleteScopeEvent.Serializer())
+               .serializer(UpdateReaderGroupEvent.class, 14, new UpdateReaderGroupEvent.Serializer());
```

Comment:
Please do not change the serializer ids. This can break backward compatability.

Reply:
The tests for DeleteScopeEvent inside ControllerEventSerializerTest are missing too.

Reply:
Thank you @shrids , I have corrected the sequence and added a test too